.changes/v3.6.0/816-improvements.md

```diff
@@ -0,0 +1,4 @@
+* `resource/vcd_network_isolated_v2` add support for `metadata` so this field can be set when creating/updating isolated networks. [GH-816]
```

Comment:
Looks like docs for the new fields are missing?

Reply:
Thanks for noticing, I need to be more careful, I also forgot the entries of #804. I've added them all.